generic_config_updater/services_validator.py

```diff
@@ -101,3 +101,20 @@ def caclmgrd_validator(old_config, upd_config, keys):
 
 def ntp_validator(old_config, upd_config, keys):
     return _service_restart("ntp-config")
+
+def vlanintf_validator(old_config, upd_config, keys):
+    old_vlan_intf = old_config.get("VLAN_INTERFACE", {})
+    upd_vlan_intf = upd_config.get("VLAN_INTERFACE", {})
+
+    old_keys = [ tuple(key.split("|"))
```

Comment:
>old_keys [](http://example.com/codeflow?start=4&length=8)

Could you add some comments into each code block? For example, give some sample of old_keys and upd_keys, and explain your intention.